src/pymatgen/io/jdftx/README.MD

```diff
@@ -0,0 +1,35 @@
+# JDFTx io
```

Comment:
I would suggest not putting this in a separate README, but rather in the actual `__init__.py` file. Speaking of which, you need to add a `__init__.py` for this module. There is no way this code would actually be properly imported.

Reply:
Should be fixed in my review_edits branch - let me know when you are done reviewing master in its current state and I will merge my new changes to master for this fork. Also thank you for taking the time to review this!

Reply:
Sorry for confusion - wasn't sure if it would be cleaner to wait until you finished or to merge fixes as we went. Fixes to your comments are now merged to master (pinging for re-review just in case)